### ISSUE TITLE
Showing articles/save button/clear button are not aligned correctly (Flare)

### ISSUE THREAD

**Issue by luchia** [TeamMentor/TM_4_0_Design#292]
![screen shot 2015-01-22 at 12 01 40 pm](https://cloud.githubusercontent.com/assets/4547737/5849230/713a091c-a22e-11e4-846e-f60c745eef4f.png)

**Comment by luchia** [comment]
[This has been fixed](https://github.com/TeamMentor/TM_4_0_Design/commit/ac8673cbd3b3bf672b1ede168114bdc6ec7be27b).

![screen shot 2015-01-23 at 10 45 11 am](https://cloud.githubusercontent.com/assets/4547737/5867918/e115fd62-a2ed-11e4-82d4-161b278e2d08.png)
![screen shot 2015-01-23 at 10 45 14 am](https://cloud.githubusercontent.com/assets/4547737/5867917/e1136b38-a2ed-11e4-8e4d-b381b055257e.png)
![screen shot 2015-01-23 at 10 47 31 am](https://cloud.githubusercontent.com/assets/4547737/5867919/e118db86-a2ed-11e4-9daf-0fef546eeda3.png)

**Comment by luchia** [comment]
Further work on the links as well, I decided to change them to buttons because they will post data and it's more obvious for the user. Left is unhovered state, right is hovered state -

![screen shot 2015-01-23 at 11 27 01 am](https://cloud.githubusercontent.com/assets/4547737/5868318/552f1896-a2f3-11e4-8e63-3d7bab962000.png)

**Comment by luchia** [comment]
Wasn't happy with the alignment on mobile so I changed the structure over to columns and it looks a lot better. I also removed the words because no matter what way I do it, the clear button always breaks to two lines before the save one does.
My thoughts are that we will have already primed our users for what these buttons do when we release Jade as a beta for TM 4.0. By the time we roll out Flare, they'll understand what those buttons do and if they're new users, our user onboarding will teach them how to use the application. Besides that, the icons are clear as to what they will do.

NB: Colours are off because it's a GIF, check it out on the Dev branch to see it in the flesh.

![resize-main 1](https://cloud.githubusercontent.com/assets/4547737/5869161/23102cb2-a300-11e4-83ae-cf2b764e37f5.gif)
